ImageManager/ImageManager.py

```diff
@@ -40,10 +52,25 @@ def get_image(imageid):
             return format_response(e.error_code, e.message)
 
 
+@image.route('/image/<imageid>/binary', methods=['GET'])
+def get_image_binary(imageid):
+    try:
+        init_tenant_context(request, db, minioClient)
+        orm_image = assert_image_exists(imageid)
+        filename = imageid + '.hex'
+        return send_from_directory(directory='/tmp/', filename=filename)
```

Comment:
Here we're actually returning the "cache" copy used when the user uploads to minio?

Reply:
Fixed